src/engine.rs

```diff
@@ -318,6 +326,16 @@ impl Engine {
     }
 }
 
+#[cfg(feature = "profile_eth_gas")]
+fn log_eth_gas_used(executor: &StackExecutor<MemoryStackState<Engine>>) {
+    use alloc::format;
+    sdk::log(format!(
+        "{}: {}",
+        crate::prelude::ETH_GAS_USED,
+        executor.used_gas()
+    ));
```

Comment:
Feels a bit strange that it's `pub fn log(data: String) {` rather than `pub fn log(data: &str) {`

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/41/commits/58e0cc479a92e2aabd6beadf27087a5c5fef74eb